### DISCUSSION TITLE
How to run PBPK model to simulate drug concentration of a real patient (using CLI)

### DISCUSSION THREAD

**Discussion by lamnvcnsh** [Open-Systems-Pharmacology/Forum#844] [Questions & Problems] [answered]
Dear everyone,

I would like to ask you a question related to running simulation using CLI as an example bellow.

1. I downloaded a PBPK project (efavirenz from OPS-PBPK-Model-Library), in this project, I can see a few pre-defined individuals based on previous papers.
2. In the graphic user interface (GUI), I could create a new individual (such as real individual information), then I could run a simulation for this individual. 
3. My question is how I can do simulation and get the results with CLI instead of GUI if I have a PBPK model, pre-defined protocol (for simulation), and individual information.
4. If yes, how can I prepare as input data for simulation?

I am a newbie, so sorry if it is a silly question.
Thank you very much.

**Comment by msevestre** [comment, marked as answer]
You should use the ospsuite-r for this.
The CLI is used to run preexisting projects or snapshots but there is no scripting capabilities.

See the OSPSuite-r  repository for more information on setup and getting started

**Comment by lamnvcnsh** [comment]
Thank @msevestre so much for your information.
I will try with OSPSuite-r, then let you known if I have any issue.

**Comment by msevestre** [comment]
Sounds good. Good luck @lamnvcnsh
